lib/ES/Util.pm

```diff
@@ -180,6 +267,19 @@ sub _check_build_error {
     warn join "\n", ( '', @warn, '' );
 }
 
+#===================================
+sub _xml_lint {
+#===================================
+    my ( $dest ) = @_;
+    return run(
+            'xmllint',
```

Comment:
Calling `run` (or any subprocess work, really) probably always warrants error handling and reporting and probably even vigorous dying, in the event the world happened to be on fire that day.